src/noc/store/actions/noc.action.ts

```diff
@@ -1,6 +1,9 @@
 import { Action } from '@ngrx/store';
-import { NocState, NocError, NocQuestion, NoCAnswer, NocStateData, NocEvent } from '../../models/noc.state';
-
+import { NocAnswer } from '../../models/noc-answer.interface';
```

Comment:
could you not add an index.ts which re-exports these, then do a barrel import here?